buildpipeline/pipelines.json

```diff
@@ -425,7 +438,42 @@
             "Type": "build/product/",
             "PB_BuildType": "Release"
           }
-        }
+        },
```

Comment:
Thanks @wtgodbe . I've changed the values of "Rid" and "TestContainerSuffix" as you suggested.